src/main/java/com/commercetools/sync/producttypes/utils/AttributeDefinitionUpdateActionUtils.java

```diff
@@ -18,22 +27,131 @@
      * and the {@link AttributeDefinitionDraft} have identical fields, then no update action is needed and hence an
      * empty {@link List} is returned.
      *
-     * @param oldAttributeDefinition        the attribute definition which should be updated.
-     * @param newAttributeDefinitionDraft   the attribute definition draft where we get the new fields.
-     * @param syncOptions                   responsible for supplying the sync options to the sync utility method.
-     *                                      It is used for triggering the error callback within the utility, in case of
-     *                                      errors.
+     * @param oldAttributeDefinition      the attribute definition which should be updated.
+     * @param newAttributeDefinitionDraft the attribute definition draft where we get the new fields.
      * @return A list with the update actions or an empty list if the attribute definition fields are identical.
-     *
      */
     @Nonnull
     public static List<UpdateAction<ProductType>> buildActions(
         @Nonnull final AttributeDefinition oldAttributeDefinition,
-        @Nonnull final AttributeDefinitionDraft newAttributeDefinitionDraft,
-        @Nonnull final ProductTypeSyncOptions syncOptions) {
+        @Nonnull final AttributeDefinitionDraft newAttributeDefinitionDraft) {
```

Comment:
no unit tests for this method?

Reply:
I haven't added a unit test for this method. I added a test in the integration tests, since this is a method that calls to other methods that already have its own unit test.